includes/integration.php

```diff
@@ -22,11 +23,11 @@ class Metrilo_Integration extends WC_Integration
             'checkout_start' => 'Started Checkout',
             'identify' => 'Identify calls'
         );
-        private $plugin_log_path = METRILO_ANALYTICS_PLUGIN_PATH . 'Metrilo_Analytics.log';
+        private $plugin_log_name = 'Metrilo_Analytics.log';
```

Comment:
Why did you do it like that? You are using `plugin_log_name` only in a combination with `METRILO_ANALYTICS_PLUGIN_PATH` .  Better keep the `plugin_log_path` variable.

Reply:
Made this change because initially the plugin path constant was part of the main plugin class and on a later stage it was moved to this integration classs and added with a wordpress hook. Since this hook is created after the private class variable I removed the path from the variable and added it directly to the method responsible for the logging of the error.